packages/react-chat/src/components/Chat/index.tsx

```diff
@@ -63,15 +70,33 @@ const Chat: React.FC<ChatProps> = ({
   const dialogRef = useRef<HTMLElement>(null);
   const [hasAlert, setAlert] = useState(false);
 
+  const { config } = useContext(RuntimeStateAPIContext);
+  const state = useContext(RuntimeStateContext);
+
+  useEffect(() => {
+    if (autostart) onStart?.();
+  }, [autostart]);
```

Comment:
I think we should just do this in `useRuntimeState` so it is much more centralized, rather than within a component?